web/home/src/pages/Home/home.tsx

```diff
@@ -0,0 +1,152 @@
+import {  useEffect, useState } from 'react';
+
+import massaLogomark from '../../assets/massa_logomark_detailed.png';
+
+import axios from 'axios';
+import { PluginHomePage } from '../../../../shared/interfaces/IPlugin';
+import { PluginCard } from '../../components/pluginCard';
+
+import Header from '../../components/Header';
+
+import ManagePluginCard from '../../components/managePluginCard';
+import grid1 from '../../assets/element/grid1.svg'
+import wallet from '../../assets/logo/plugins/Wallet.svg'
+import registry from '../../assets/logo/plugins/Registry.svg'
+import webOnChain from '../../assets/logo/plugins/WebOnChain.svg'
+import MainTitle from '../../components/MainTitle';
+
+/**
+ * Homepage of Thyra with a list of plugins installed
+ *
+ */
+type Props = {};
+
+function Home(props: Props) {
+  // Fetch plugins installed by calling get /plugin/manager
+  
+  const fakePluginsList:PluginHomePage[] = [
+    {
+      name: "Massa's Wallet",
+      description:
+      'Create and manage your smart wallets to buy, sell, transfer and exchange tokens',
+      id: '420',
+      logo: wallet,
```

Comment:
what is the type of variable `wallet` here? how the plugin is supposed to pass the logo?